editor/src/components/canvas/canvas-globals.spec.ts

```diff
@@ -138,12 +138,25 @@ const selectorPropertyControlsInfo: PropertyControlsInfo = {
   },
 }
 
+const otherCardComponentDescriptorWithName: ComponentDescriptorWithName = {
+  ...cardComponentDescriptor,
+  componentName: 'Other Card',
+}
+
+const otherCardControlsToCheck: ControlsToCheck = Promise.resolve(
+  right([otherCardComponentDescriptorWithName]),
+)
+
 describe('validateControlsToCheck', () => {
   beforeEach(() => {
     // Twice because the first will leave some data in `previousModuleNamesOrPaths`.
     resetControlsToCheck()
     resetControlsToCheck()
   })
+  afterAll(() => {
+    resetControlsToCheck()
+    resetControlsToCheck()
+  })
```

Comment:
Why the `afterAll` as well now?

Reply:
Just means any subsequent test in the future wont be caught out by this having some data kicking around.